src/interfaces/IMorpho.sol

```diff
@@ -112,6 +112,8 @@ interface IMorphoBase {
     /// on `transfer` and `transferFrom`. In particular, tokens with fees on transfer are not supported.
     /// - The IRM should not re-enter Morpho.
     /// - The oracle should return a price with the correct scaling.
+    /// - The oracle should not be able to go down instantly by a multiplier of less than LLTV*LIF. In particular, if
```

Comment:
"go down by a multiplier" is less clear now (although correct!), due to the possible confusion between decreasing by a factor vs. being multiplied by 1/factor.

```suggestion
    /// - The oracle should not change instantly such that the new price is less than the old price times LLTV*LIF. In particular, if
```